addons/point_of_sale/static/tests/tours/utils/product_screen_util.js

```diff
@@ -660,3 +660,27 @@ export function addDiscount(discount) {
             .flatMap((key) => Numpad.click(key)),
     ].flat();
 }
+
+export function checkVariantProductInfo(titel) {
+    //check if title and price and inventory section are present
+    return [
+        {
+            content: "Check if modal title contains title",
+            trigger: ".modal-title.text-break.flex-grow-1:contains('Office Combo')",
+            run: function () {
+                const titleElement = document.querySelector(".modal-title.text-break.flex-grow-1");
+                const titleText = titleElement.textContent.trim();
+                return titleText.includes(titel);
```

Comment:
Can we skip trimming with `trim`? Since we're checking with `includes` anyway, so even if there are whitespaces, it finds the text if it exists 👀 

```suggestion
                return document.querySelector(".modal-title.text-break.flex-grow-1").textContent.includes(title);
```